apps/server/src/shared/domain/entity/course.entity.ts

```diff
@@ -115,15 +138,7 @@ export class Course extends BaseEntityWithTimestamps implements ILearnroom, IEnt
 		};
 	}
 
-	getNumberOfStudents(): number {
```

Comment:
At the moment it is not used. But it is possible to use it. 
At the moment we have one place submission.coursegroup where we need the concret Ids instead of the number. 
Also i have implement the logic for course, coursegroup and lesson that can pull it both. 

But for the task it self we only need the number. I am not happy with the solution that we replace max number return value with userIds. But implementing both operations for course, lesson and coursegroup it looks overkill. 

If we can avoid the need of the coursegroup in the submission in future we can simplify a lot of logic. 
getStudentIds  -> getNumberOfStudents 

But for now i use getStudentIds().length to get the number. Is less code ...but for me it looks like a hacky solution. :/

Reply:
Alright, I saw that too reading further.